Ginger/GingerCoreCommon/ReporterLib/UserMsgsPool.cs

```diff
@@ -321,15 +322,17 @@ public static void LoadUserMsgsPool()
             Reporter.UserMsgsPool.Add(eUserMsgKey.SourceControlUpdateFailed, new UserMsg(eUserMsgType.ERROR, "Update Solution", "Failed to update the solution from source control." + Environment.NewLine + "Error Details: '{0}'", eUserMsgOption.OK, eUserMsgSelection.None));
             Reporter.UserMsgsPool.Add(eUserMsgKey.SourceControlCommitFailed, new UserMsg(eUserMsgType.ERROR, "Commit Solution", "Failed to commit the solution from source control." + Environment.NewLine + "Error Details: '{0}'", eUserMsgOption.OK, eUserMsgSelection.None));
             Reporter.UserMsgsPool.Add(eUserMsgKey.SourceControlChkInSucss, new UserMsg(eUserMsgType.QUESTION, "Check-In Changes", "Check-in process ended successfully." + Environment.NewLine + Environment.NewLine + "Do you want to do another check-in?", eUserMsgOption.YesNo, eUserMsgSelection.No));
-            Reporter.UserMsgsPool.Add(eUserMsgKey.SourceControlChkInConflictHandled, new UserMsg(eUserMsgType.QUESTION, "Check-In Results", "The check in process ended, please notice that conflict was identified during the process and may additional check in will be needed for pushing the conflict items." + Environment.NewLine + Environment.NewLine + "Do you want to do another check-in?", eUserMsgOption.YesNo, eUserMsgSelection.No));
+            Reporter.UserMsgsPool.Add(eUserMsgKey.SourceControlChkInConflictHandled, new UserMsg(eUserMsgType.QUESTION, "Check-In Results", "The check in process ended, please notice that conflict was identified during the process and may additional check in will be needed for pushing the conflict items." + Environment.NewLine + Environment.NewLine + "Do you want to do another check-in?", eUserMsgOption.YesNo, eUserMsgSelection.No));   
```

Comment:
Actually this message needs to be removed, Jinendra said this message doesn't makes sense when the current check-in process has completed, so don't show this message. so removing this message.
In case of Local Commit we don't show any success message, we close the window.